src/DataFixtures/AppFixtures.php

```diff
@@ -9,19 +9,26 @@
 use Faker\Factory; // Using fakerphp/faker
 use DateTime;
 use DateTimeImmutable;
+use Doctrine\ORM\EntityManagerInterface;
 
 class AppFixtures extends Fixture
 {
     private UserPasswordHasherInterface $passwordHasher;
+    private EntityManagerInterface $entityManager;
 
-    public function __construct(UserPasswordHasherInterface $passwordHasher)
+    public function __construct(UserPasswordHasherInterface $passwordHasher, EntityManagerInterface $entityManager)
```

Comment:
make it inline in constructer for proprety declaration. Since you updated to php 8.3 please use readony class, as mentioned here https://stitcher.io/blog/readonly-classes-in-php-82